openspec/changes/add-grill-for-unknowns-agent-skill/tasks.md

```
@@ -0,0 +1,22 @@
+## 1. Canonical skill and provenance
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)